modules/test/playwright/tests/frontend-data-set-views-web/creationActions.spec.ts

```diff
@@ -618,6 +625,53 @@ fragmentTest.describe('Creation Actions in the fragment', () => {
 
 				await expect(dialog).not.toBeInViewport();
 			});
+
+			await fragmentTest.step(
+				'Creation Action of type "modal" with custom header (Commerce) does not duplicate the title',
+				async () => {
+					await commerceAdminProductPage.goto();
```

Comment:
Shouldn't this test be in a commerce project, `/tests/commerce/something`? It's not strictly related to DSM creation actions, right? Just to FDS modifications.

Reply:
I'm in doubt here, but as it is related with the way FDS actions handle the modal/panel, in any place they appear. 
Without specific FDS tests, here we have the contrast between FDS with custom vs without custom headers.